goroutine.go

```diff
@@ -407,11 +407,12 @@ func (gd *GoroutineDump) withCondition(cond string, callback func(int, *Goroutin
 	goroutines := make([]*Goroutine, 0, len(gd.goroutines))
 	for i, g := range gd.goroutines {
 		params := map[string]interface{}{
-			"id":       g.id,
-			"duration": g.duration,
-			"lines":    g.lines,
-			"state":    g.metas[MetaState],
-			"trace":    g.trace,
+			"id":         g.id,
+			"duration":   g.duration,
+			"lines":      g.lines,
+			"state":      g.metas[MetaState],
+			"trace":      g.trace,
+			"duplicates": len(g.duplicates),
```

Comment:
Better move up above the "duration" line.

Reply:
I have moved it up. Willing to make it dup (or dups) if you prefer that.